apps/ui/src/components/SpaceBilling.vue

```diff
@@ -0,0 +1,165 @@
+<script setup lang="ts">
+import dayjs from 'dayjs';
+import { getGenericExplorerUrl } from '@/helpers/generic';
+import { _n } from '@/helpers/utils';
+import { getNetwork } from '@/networks';
+import { usePaymentsQuery } from '@/queries/payments';
+import { Space } from '@/types';
+
+const props = defineProps<{ space: Space }>();
+
+const network = computed(() => props.space.network);
+const chainId = computed(() => getNetwork(network.value).chainId);
```

Comment:
All the computed definitions should be together, after the composable declarations